tfe/resource_tfe_organization_token.go

```diff
@@ -0,0 +1,84 @@
+package tfe
+
+import (
+	"fmt"
+	"log"
+
+	tfe "github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceTFEOrganizationToken() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceTFEOrganizationTokenCreate,
+		Read:   resourceTFEOrganizationTokenRead,
+		Delete: resourceTFEOrganizationTokenDelete,
+
+		Schema: map[string]*schema.Schema{
+			"organization": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"token": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
```

Comment:
I believe this wants `Sensitive: true`?

Reply:
Good point!

Fixed in the next commit.